tests/fuzz/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "typst-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+arbitrary = { version = "1.3.1", features = ["derive"] }
+libfuzzer-sys = "0.4"
+typst-syntax = { path = "../../crates/typst-syntax" }
+typst-library = { path = "../../crates/typst-library" }
+typst-render = { path = "../../crates/typst-render" }
+typst = { path = "../../crates/typst" }
+comemo = "0.3"
+unscanny = "0.1"
+
+[profile.release]
+debug = 1
```

Comment:
This profile doesn't seem to be used due to workspace config. Is that a problem?

Reply:
I'm not sure what you mean by the workspace config. But the number of iterations that a fuzzer can go through is important in terms of fuzzing performance. Because of this it's usually the case that you'd build the fuzz harness in release mode. However in release mode certain checks are turned off by default (e.g. integer additions aren't checked for overflows, like they are in debug mode). As these (integer overflows) are typically bugs we want to enable these checks during fuzzing. So I will often use this profile when fuzzing.

Reply:
If you build it, cargo will print a warning that this release profile is ignored. It only considers profiles in the workspace root. However, there we don't want to enable debug=1 because it would affect the other targets. Maybe there could be a special fuzzing profile, not sure how that'd work with cargo fuzz.

Reply:
This is the default profile from cargo-fuzz. It was generated via `cargo fuzz init`, so from that perspective I think it's fine. But I'll investigate a little further how this fit's within the larger cargo workspace.

Reply:
cargo fuzz's default profile doesn't assume that it's in a workspace, so it would be fine then.